userprofile/views.py

```diff
@@ -23,38 +28,71 @@
 from .models import Category, Profile, Skill, TermsOfService
 
 
-class ProfileListView(ListView):
+def _get_user_profiles_from_search(users: [User], search: str):
+    """
+    Determine full name similarity for each user, and return a list of corresponding profiles, ordered by best match
+    """
+    search_matches = []
+    for user in users:
+        match_score = fuzz.token_set_ratio(user.get_full_name(), search)
+        if match_score > 10:
+            search_matches.append((match_score, user))
+    # Order by best match, descending
+    search_matches.sort(key=lambda t: t[0], reverse=True)
+    # Ordered profiles
+    return [s[1].profile for s in search_matches]
+
+
+class MembersAPIView(APIView):
+    """
+    API view returning an HTML response containing paginated profiles
+    based on member name search
+    """
+
```

Comment:
Denne tomme linjen er unødvendig eller det burde være tom linje på samme sted i metoden over. Samme for meg, men bør være konsekvent.